chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/QuestionableActivityManager.java

```diff
@@ -212,17 +191,6 @@ private boolean isSvapAllowedForCriteria(CertificationResult certResult) {
         return certResult.getAllowedSvaps() != null && certResult.getAllowedSvaps().size() > 0;
     }
 
-    private void createListingActivity(QuestionableActivityListingDTO activity, Long listingId, Date activityDate,
```

Comment:
What happened with this method? I don't see it recreated anywhere; was it leftover and unused?